### ISSUE TITLE
Keyboard and visual users are unaware of the shortcut key to navigate in 'Text view'.

### ISSUE THREAD

**Issue by msftedad** [microsoft/monaco-editor#4222]
### Reproducible in vscode.dev or in VS Code Desktop?

- [X] Not reproducible in [vscode.dev](https://vscode.dev) or VS Code Desktop

### Reproducible in the monaco editor playground?

- [ ] Not reproducible in [the monaco editor playground](https://microsoft.github.io/monaco-editor/playground.html)

### Monaco Editor Playground Link

[Monaco Editor (microsoft.github.io)](https://microsoft.github.io/monaco-editor/playground.html?source=v0.42.0-dev-20230901#XQAAAAJiAgAAAAAAAABBqQkHQ5NjdMjwa-jY7SIQ9S7DNlzs5W-mwj0fe1ZCDRFc9ws9XQE0SJE1jc2VKxhaLFIw9vEWSxW3yscw1yeqRgmSaumWUF-TBOsVJ4S4FwfCDQb5-2uUHXTVAefbVSd9IzaZ_vv9VR9KugsKi2Ej5cnSVboBg9CyozatOzDH3E9tWPTZl_NBNEBvi0EHoUzILAf1hALFydK2aeQUqERz5WxlOM79ePP0z_yCANpSkEw04zalxBSQbh-1_S1xj2LMDTy-gO5QA7HukP82J5yZZlhrb1rjcJMe1UuAX8vk8ijuR-nQfRJBlFprFc4W92jtJP-VIgFMOhJMlzNNfDzQo2b1GFfX9JxTj9AWbttIJzS0cGw-5lkj__o9Meo)

### Monaco Editor Playground Code

```typescript
const value = /* set from `myEditor.getModel()`: */ `test test test test test test test test test test test test test test test test test test test test test test test test test test test test test test test test test test test test test test test test test test test test test test test test test test test test test test `;

// Hover on each property to see its docs!
const myEditor = monaco.editor.create(document.getElementById("container"), {
	value,
	language: "javascript",
	automaticLayout: true,
	readOnly: true,
	wordWrap: "on",
});
```

### Reproduction Steps

1. Open the  URL- [Monaco Editor (microsoft.github.io)](https://microsoft.github.io/monaco-editor/playground.html?source=v0.42.0-dev-20230901#XQAAAAJiAgAAAAAAAABBqQkHQ5NjdMjwa-jY7SIQ9S7DNlzs5W-mwj0fe1ZCDRFc9ws9XQE0SJE1jc2VKxhaLFIw9vEWSxW3yscw1yeqRgmSaumWUF-TBOsVJ4S4FwfCDQb5-2uUHXTVAefbVSd9IzaZ_vv9VR9KugsKi2Ej5cnSVboBg9CyozatOzDH3E9tWPTZl_NBNEBvi0EHoUzILAf1hALFydK2aeQUqERz5WxlOM79ePP0z_yCANpSkEw04zalxBSQbh-1_S1xj2LMDTy-gO5QA7HukP82J5yZZlhrb1rjcJMe1UuAX8vk8ijuR-nQfRJBlFprFc4W92jtJP-VIgFMOhJMlzNNfDzQo2b1GFfX9JxTj9AWbttIJzS0cGw-5lkj__o9Meo)
2. Monaco Editor page will appear navigate inside 'Preview' Section.
3. Navigate to source code and observe the issue whether there is any information provided to access the tooltip or not.

### Actual (Problematic) Behavior

Keyboard and visual users are unaware of the shortcut key to navigate in 'Text view'.
Here when the screen reader focus on source code the screen reader announces the short cuts keys information, but the information of the tooltip short cut is not visually displayed on the screen.
Here screen reader is announcing as "Editor content; Press Alt+F1 for Accessibility Options. edit has auto complete multi line". 

Attachment:

https://github.com/microsoft/monaco-editor/assets/93735775/91f12eb2-a398-4da1-b68c-780887a5bceb

<img width="615" alt="Keyboard and visual users are unaware of the shortcut key to navigate in 'Text view'" src="https://github.com/microsoft/monaco-editor/assets/93735775/583e7d3a-b8fb-4b29-8f34-d007af4d34de">

### Expected Behavior

The information of the tooltip short cut should be visually displayed on the screen. |

**Comment by msftedad** [comment]
Hi @hediet, is any update on this issue? could you please provide the ETA to fix this issue?

**Comment by msftedad** [comment]
Hi @rzhao271, is any update on this issue? could you please provide the ETA to fix this issue?

**Comment by msftedad** [comment]
Hi @meganrogge, is any update on this issue? could you please provide the ETA to fix this issue?

**Comment by meganrogge** [comment]
Hi, this was fixed a long time ago. Check the latest build.
